app/views/users/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+  <%# form_tag("/users", method: "post", remote: true) do %> 
+  <%# label_tag 'username' %> 
+      <%# text_field(:user, :username)%>
+  </br>
+  <%# label_tag "password" %> 
+  </br>
+  <%# text_field(:user, :password)%>
+  </br>
+  <%# label_tag "email" %> 
+  </br>
+  <%# text_field(:user, :email)%>
+  </br>
+  <%# submit_tag("Submit") %> 
+  <%# end %> 
+
+  <form action="/users" method="post" accept-charset="UTF-8" data-remote="true">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="password">Password:</label>
+  <input type="text" id="password" name="user[password]"><br><br>
+  <label for="email">Email:</label>
+  <input type="Email" id="email" name="user[email]"><br><br>
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="submit" value="Submit">
+  </form>   
+  
+<%= form_with scope: :user, url: users_path do |form| %>
+  <%=label(:user, :username, "Your username :", value: "test")%>
+  <%= form.text_field :username %><br />
+  <%= form.label :password %>:
+  <%= form.text_field :password %><br />
+  <%= form.label :email %>:
+  <%= form.text_field :email, placeholder: "a@gmail.com"%><br />
+  <%= form.submit "Submit" %>
+<% end %>
```

Comment:
1. In line 39 change this line to `form_for @user do |form|` to make use of `@user`. with `form_with` you will have an error.
2. In line 43 instead of `form.text_field` use `form.password_field`
2. In line 45 instead of `form.text_field` use `form.email_field`